os-autoinst-distri-opensuse-deps.spec

```diff
@@ -77,6 +77,7 @@ Requires:       perl(XML::Writer)
 Requires:       perl(YAML::PP)
 # END AUTOGENERATED DEPENDENCY LIST
 Recommends:     os-autoinst-devel
+Requires:       golang-github-google-jsonnet
```

Comment:
Because this is quite special please add a comment and also reference a ticket to explain this

Reply:
Done